submissions/the-darkesttt/memory-pair-game/js/newApp.js

```diff
@@ -0,0 +1,209 @@
+let stopGame = false;
+let foundedPairs = 0;
+let flippedCardsArr = [];
+let cardsItems = '';
+let min = 0;
+let sec = 0;
+
+const body = document.querySelector('body');
+const gameList = document.querySelector('.game-list');
+
+const gameMenu = document.querySelector('.game-menu');
+const gameMenuTitle = document.querySelector('.modal__title');
+const gameMenuDecs = document.querySelector('.modal__desc');
+const gameStartButton = document.querySelector('#game-start-button');
+
+const restartBtnModal = document.createElement('button');
+restartBtnModal.innerHTML = 'Restart';
+restartBtnModal.classList.add('restart-btn-modal');
+
+const timerMin = document.querySelector('#timerMin');
+const timerSec = document.querySelector('#timerSec');
+
+function hideMenu() {
+  body.classList.add('menu-close');
+  gameMenu.classList.add('game-menu__hide');
+  setTimeout(countGameTime, 1000)
+  
+}
+
+gameStartButton.addEventListener('click', hideMenu);
+restartBtnModal.addEventListener('click', () => {
+  restartingGame();
+  timerSec.innerHTML = `00`;
+  timerMin.innerHTML = `00`;
+  hideMenu();
+});
+
+function countGameTime() {
+  if (!stopGame) {
+    sec++;
+
+    if (sec < 10) {
+      timerSec.innerHTML = `0${sec}`;
+    } else {
+      timerSec.innerHTML = sec;
+    }
+
+    if (min < 10) {
+      timerMin.innerHTML = `0${min}`;
+    } else {
+      timerMin.innerHTML = min;
+    }
+
+    if (sec === 59) {
+      sec = 0;
+      min++;
+    }
+
+    setTimeout(countGameTime, 1000);
+  }
+}
+
+const restartButton = document.querySelector('#restart-btn');
```

Comment:
Move all constants and variable declarations to the top of the file. if they are on the root level.